src/main/java/org/threadly/concurrent/wrapper/intercepter/PrioritySchedulerTaskIntercepter.java

```diff
@@ -10,23 +11,37 @@
 
 /**
  * <p>Class to wrap {@link PrioritySchedulerService} pool so that tasks can be intercepted and either 
- * wrapped, or modified, before being submitted to the pool.  This abstract class needs to have 
- * {@link #wrapTask(Runnable, boolean)} overridden to provide the task which should be submitted to the 
- * {@link PrioritySchedulerService}.  Please see the javadocs of {@link #wrapTask(Runnable, boolean)} for 
- * more details about ways a task can be modified or wrapped.</p>
+ * wrapped, or modified, before being submitted to the pool.  This class can be passed a lambda to
+ * {@link #PrioritySchedulerTaskIntercepter(PrioritySchedulerService, BiFunction)}}, or 
+ * {@link #wrapTask(Runnable, boolean)} can be overridden to provide the task which should be submitted 
+ * to the {@link PrioritySchedulerService}.  Please see the javadocs of 
+ * {@link #wrapTask(Runnable, boolean)} for more details about ways a task can be modified or 
+ * wrapped.</p>
  * 
  * <p>Other variants of task wrappers: {@link ExecutorTaskIntercepter}, 
  * {@link SubmitterSchedulerTaskIntercepter}, {@link PrioritySchedulerTaskIntercepter}.</p>
  * 
  * @author jent - Mike Jensen
  * @since 4.6.0
  */
-public abstract class PrioritySchedulerTaskIntercepter extends SchedulerServiceTaskIntercepter 
+public class PrioritySchedulerTaskIntercepter extends SchedulerServiceTaskIntercepter 
                                                        implements PrioritySchedulerService {
```

Comment:
nit, but would you re-align the `implements` when it has to wrap on the next line.  The `i` of implements should be under the `e` of extends.